Jenkinsfile

```diff
@@ -0,0 +1,131 @@
+@Library('xmos_jenkins_shared_library@v0.27.0')
+
+def runningOn(machine) {
+  println "Stage running on:"
+  println machine
+}
+
+getApproval()
+pipeline {
+  agent none
+
+  parameters {
+    string(
+      name: 'TOOLS_VERSION',
+      defaultValue: '15.2.1',
+      description: 'The XTC tools version'
+    )
+    booleanParam(
+      name: 'XMATH_SMOKE_TEST',
```

Comment:
how long the tests last if XMATH_SMOKE_TEST is set to False? I wonder we should use it by default if they last less than 30 min. If we decide to run the tests in smoke test mode, when do we run the full tests?